packages/p2p/src/stores/sendbird-store.ts

```
@@ -163,7 +163,7 @@ export default class SendbirdStore extends BaseStore {
                 await this.initialiseOrderChannel();
             }
         } catch (error) {
-            this.setChatError();
+            // handle sendbird error
```

why remove this.setChatError?

---

Same reason as above @ameerul-deriv

---

but we shouldn't ignore the error right? we should still display it IMO, or is it causing a huge issue every time we chat with someone

---

Would you guys rather I restore the `console.warn` because those are also not allowed in the code base are they?

---

we don't have to show it as a console.warn, but if we dont show any error and the user is communicating with an advertiser, and then theres a chat error, how would the user know?